ethcore/light/src/net/mod.rs

```diff
@@ -157,6 +158,54 @@ pub struct Peer {
 	awaiting_acknowledge: Option<(SteadyTime, Arc<FlowParams>)>,
 }
 
+/// Whether or not a peer was kept by a handler
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum PeerStatus {
```

Comment:
Because by reading the signature there is no way to know if `true` means kept or dropped. If it was called `is_dropped` I would return a `bool`. I don't know about you, but I hit "what does this bool _even mean_" all the time so I try to make sure my code doesn't fall into the same trap.